image/templates/helm/shared/templates/02-scanner-v4-06-db-pvc.yaml

```diff
@@ -24,7 +24,6 @@ metadata:
     {{- include "srox.labels" (list . "persistentvolume" $pvName) | nindent 4 }}
   annotations:
     {{- include "srox.annotations" (list . "persistentvolume" $pvName) | nindent 4 }}
-    "helm.sh/hook": "pre-install,pre-upgrade"
     "helm.sh/resource-policy": keep
```

Comment:
I have a question about the `resource-policy: keep` semantics. According to the Helm documentation a resource with `resource-policy: keep` becomes orphaned when uninstalling a chart. Does this also mean, that when (re-)installing the chart again, this resource will be adopted again? Will a pre-existing PVC with the expected name be used automatically by scanner V4?

Reply:
Correct, helm will leave the PVC there. For reinstall it will then work with the same PVC. (worked for my test)

I'm not sure though if that resource is then part of the helm lifecycle or not. All I can say is that opposed to the pre-hook solution, helm does not have errors reinstalling when that PVC already exists.

Reply:
As discussed in today's standup I've removed that annotation as well.